<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartForm</class>
 <widget class="QWidget" name="ChartForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,40,0,0" columnstretch="0,20,0,0">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="0" column="3">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>45</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="3">
      <widget class="QSlider" name="vs">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSlider" name="hs">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" rowspan="3" colspan="3">
      <widget class="QWidget" name="wChart" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Управление координатной плоскостью</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="rbAuto">
          <property name="text">
           <string>Автоматическое</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="rbManual">
            <property name="text">
             <string>Вручную</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbMaxX">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbMaxX">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbMaxY">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbMaxY">
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
